packages/react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -85,6 +85,17 @@ export const Tooltip: React.FC<ITooltipProps> = (props) => {
     placement: placement,
   });
 
+  const setVisibilityWithCallback = (
```

Comment:
How about a bit different name?
```suggestion
  const handleVisibilityChange = (
```

Reply:
yep, looks nice 👍